src/mjlab/tasks/velocity/velocity_env_cfg.py

```diff
@@ -252,7 +195,7 @@ def create_velocity_env_cfg(
       weight=1.0,
       params={
         "std": math.sqrt(0.2),
-        "asset_cfg": SceneEntityCfg("robot", body_names=(viewer_body_name,)),
+        "asset_cfg": SceneEntityCfg("robot", body_names=()),  # Set per-robot.
```

Comment:
I agree aesthetically that large signatures for functions like `create_velocity_env_cfg() ` were unideal, but this "Set per-robot" as a comment does feel more brittle to me... like if the "base" config changes to require more fields that need to be overwritten, it's much easier to have silent failures